backend/app/auth.py

```diff
@@ -15,6 +15,10 @@ def verify_token(token: str) -> dict:
     keys = response.json()["keys"]
     header = jwt.get_unverified_header(token)
     key = [k for k in keys if k["kid"] == header["kid"]][0]
-    # The JWT returned from the Identity Provider may contain an at_hash, so we will disable the verify_at_hash check.
+    # The JWT returned from the Identity Provider may contain an at_hash
+    # jose jwt.decode verifies id_token with access_token by default if it contains at_hash
+    # FYI : https://github.com/mpdavis/python-jose/blob/4b0701b46a8d00988afcc5168c2b3a1fd60d15d8/jose/jwt.py#L59
+    # The access_token is not included in the front-end request header, so at_hash is ignored.
```

Comment:
「フロントエンドでつけていない」というよりかは、「アプリでアクセストークンを利用していないので、at_hashの検証をスキップしている」というテイストで書いてください！  
"Since we are not using an access token in the app,  skipping the verification of the at_hash."などでお願いします

Reply:
こちらも同じく修正いたしました！